src/atomate2/common/flows/phonons.py

```diff
@@ -150,6 +154,8 @@ class BasePhononMaker(Maker, ABC):
     kpath_scheme: str = "seekpath"
     code: str = None
     store_force_constants: bool = True
+    force_constants2file: str = None
+    calculate_pdos: bool = False
```

Comment:
Could you please make them part of "generate_frequencies_eigenvectors_kwargs" or at least the filename? Additionally, please use "create_force_constants_file" as an additional kwarg in "generate_frequencies_eigenvectors_kwargs" to write the force constant file? Otherwise, this will getvery messy. I would also rename "force_constants2file" to "force_constants_filename"